src/_P076_HLW8012.ino

```diff
@@ -55,6 +55,39 @@ unsigned int p076_hvoltage = 0;
 unsigned int p076_hpower = 0;
 unsigned int p076_hpowfact = 0;
 
+struct p076_DevicePinSettings {
+  int Id;
+  String Device_Name;
+  byte SEL_Pin;
+  byte CF_Pin;
+  byte CF1_Pin;
+  byte Current_Read;
+  byte CF_Trigger;
+  byte CF1_Trigger;
+
+};
+
+typedef struct p076_DevicePinSettings P076_DevicePinSettings;
+
+const int p076_PinSettingsCount = 10;
+P076_DevicePinSettings p076_PredefinedPinSettings[p076_PinSettingsCount] =
```

Comment:
Maybe this can be stored as `PROGMEM` ?
Like it is defined now, it takes quite a lot of memory from the sys stack (NonOS SDK stack).

Reply:
Moved to PROGMEM, now